precompiles/crowdloan-rewards/src/mock.rs

```diff
@@ -102,6 +102,7 @@ impl AddressMapping<TestAccount> for TestAccount {
 	}
 }
 
+//TODO Why is this impl necessary? I don't see it required anywhere...
```

Comment:
pallet_stacking use a `Runtime::AccountId: From<H160>` bound, but someone suggested to remove it and use `pallet_evm::Config::AddressMapping` instead since there is already a bound on `pallet_evm::Config`.